pilot/pkg/model/validation.go

```diff
@@ -1695,10 +1695,23 @@ func ValidateAuthenticationPolicy(msg proto.Message) error {
 		errs = appendErrors(errs, validateAuthNPolicyTarget(target))
 	}
 
+	jwtIssuers := make(map[string]bool)
 	for _, method := range in.Peers {
-		errs = appendErrors(errs, validateJwt(method.GetJwt()))
+		if jwt := method.GetJwt(); jwt != nil {
+			if _, jwtExist := jwtIssuers[jwt.Issuer]; jwtExist {
+				errs = appendErrors(errs, fmt.Errorf("Jwt with issuer %q already defined", jwt.Issuer))
+			} else {
+				jwtIssuers[jwt.Issuer] = true
+			}
+			errs = appendErrors(errs, validateJwt(jwt))
+		}
 	}
 	for _, method := range in.Origins {
+		if _, jwtExist := jwtIssuers[method.Jwt.Issuer]; jwtExist {
```

Comment:
Can you have the same jwt issuer in peers and origins?

Reply:
I'm not sure if it's a valid use case. If we want that, then we also need to make sure those jwt params are the same. Doable, but more complicated code and I don't want to have it yet until we have a concrete demand (we don't even have a good plan for peer using jwt yet). It's easier to relax restriction later than strengthen it.

Reply:
It will be valid use case if we allow peer to be JWT. For now, I think it is ok.